appengine/storage/app.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app_yaml]
+runtime: ruby
+vm: true
+entrypoint: bundle exec ruby app.rb -p 8080
+
+env_variables:
+  GCLOUD_PROJECT: <your-project-id>
```

Comment:
Did you try to deploy without GCLOUD_PROJECT set? I normally have to have it set to run locally, but not here for running in app engine